consensus/tests/common.rs

```diff
@@ -194,4 +194,294 @@ mod tests {
+        consensus_state.blockchain_length = Length(113267);
+        consensus_state.last_vrf_output =
+            VrfOutputTruncated::from("r0K80Xsb44NLx_pBjI9UQtt6a1N-RWym8VxVTY4pAAA=");
+        let mut prot_state = ProtocolState::default();
+        prot_state.body.consensus_state = consensus_state;
+        chain_a.push(prot_state).unwrap();
+
+        let mut chain_b = ProtocolStateChain::default();
+        let mut consensus_state = ConsensusState::default();
+        consensus_state.blockchain_length = Length(113267);
+        consensus_state.last_vrf_output =
+            VrfOutputTruncated::from("kKr83LYd7DyFupRAPh5Dh9eWM1teSEs5VjU4XId2DgA=");
+        let mut prot_state = ProtocolState::default();
+        prot_state.body.consensus_state = consensus_state;
+        chain_b.push(prot_state).unwrap();
+
+        let select_result = chain_a.select_longer_chain(&chain_b).unwrap();
+
+        let result_state = select_result.0.get(0).unwrap();
+        assert_eq!(
+            result_state.body.consensus_state.last_vrf_output,
+            VrfOutputTruncated::from("r0K80Xsb44NLx_pBjI9UQtt6a1N-RWym8VxVTY4pAAA=")
+        );
+    }
+
+    #[test]
+    #[wasm_bindgen_test]
+    fn test_select_secure_chain_short_range_fork() {
+        let constants = ConsensusConstants::from_genesis();
+        let mut chain_a = ProtocolStateChain::default();
+        let mut consensus_state = ConsensusState::default();
+        consensus_state.epoch_count = Length(5);
+        consensus_state.blockchain_length = Length(11);
+        let mut prot_state = ProtocolState::default();
+        prot_state.body.consensus_state = consensus_state;
+        chain_a.push(prot_state).unwrap();
+
+        let mut chain_b = ProtocolStateChain::default();
+        let mut consensus_state = ConsensusState::default();
+        consensus_state.epoch_count = Length(5);
+        consensus_state.blockchain_length = Length(10);
+        let mut prot_state = ProtocolState::default();
+        prot_state.body.consensus_state = consensus_state;
+        chain_b.push(prot_state).unwrap();
+
+        let mut chains = vec![];
+        chains.push(chain_b);
+        let select_result = chain_a.select_secure_chain(&chains, &constants).unwrap();
+        let result_state = select_result.0.get(0).unwrap();
+        assert_eq!(
+            result_state.body.consensus_state.blockchain_length,
+            Length(11)
+        );
+    }
+
+    #[test]
+    #[wasm_bindgen_test]
+    //Long range fork with greater relative min window density
+    fn test_select_secure_chain_long_range_fork_1() {
+        let constants = ConsensusConstants::from_genesis();
+        let mut chain_a = ProtocolStateChain::default();
+        let mut consensus_state = ConsensusState::default();
+        consensus_state.epoch_count = Length(23);
+        consensus_state.min_window_density = Length(77);
+        consensus_state.sub_window_densities = vec![
+            1.into(),
```

Comment:
All occurences of these will need to be replaced by 
```suggestion
            Length(1),
```
Sorry for the breaking change..

Reply:
Might make sense to refactor this out as a `const` or `const fn` since it appears so often